extension/packages/ponder/ponder.schema.ts

```diff
@@ -0,0 +1,12 @@
+import { createSchema } from "@ponder/core";
```

Comment:
for every import in `/ponder` package ts shows me this error. For some reason it search tsconfig on ther root folder. Is it only for me?
<img width="794" alt="image" src="https://github.com/user-attachments/assets/e6c987d1-b463-4652-83c7-a12c85c65be0">

Reply:
Weird, not showing this error to me.

Reply:
https://github.com/microsoft/vscode-eslint/issues/1170#issuecomment-2084646587 this helped to me. If we'll meet the same for other users, it's better to add it to the repo